src/test/java/com/thealgorithms/dynamicprogramming/LongestCommonSubsequenceTest.java

```diff
@@ -0,0 +1,35 @@
+package com.thealgorithms.dynamicprogramming;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterAll;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LongestCommonSubsequenceTest {
+
+    @Test
+    void getLCS(){
+        String str1 = "ABCD";
+        String str2 = "ABDC";
+        String expectedLCS = "ABD";
+        String actualLCS = LongestCommonSubsequence.getLCS(str1, str2);
+        assertEquals(expectedLCS, actualLCS);
+    }
+
+    // @Test
+    // void lcsString(){
+    //     String str1 = "ABCD";
+    //     String str2 = "ABDC";
+    //     String expectedString = "ABD";
+    //     int[][] actualString = LongestCommonSubsequence.lcsString(str1, str2, );
+    //     assertEquals(expectedString, actualString);
+    //
+    // }
+
+
+    @AfterAll
+    static void printCoverage(){
+        System.out.println(LongestCommonSubsequence.branchesReached);
+
+    }
+
+}
```

Comment:
Make sure that the coverage is on its own branch without the new tests, so that we can compare the coverage before and after the implementation of our new tests :)

Reply:
i needed to have the test in the same branch to check that it the coverage was actually working. 
I guess I could've purposefully left it out of commits. But to document our tests, we can just comment out @AfterAll to run the test without manual coverage or?

Reply:
because it's not possible to run a test on a branch that has the coverage implementation but no Test to call it?..

Reply:
good point. I'll address it

Reply:
@einbergisak removing the tests causes the printCoverage to not be run at all. 
I tried:
- removing the tests and keeping @AfterAll call
- removing the tests and renaming @AfteAll to @Test
- having a 'dummy test' dummy()
- keeping getLCS() test but it being blank, as below. 

```java
    @Test
    void getLCS(){
        String str1 = "ABCD";
        String str2 = "ABDC";
        String expectedLCS = "ABD";
        String actualLCS = LongestCommonSubsequence.getLCS(str1, str2);
        assertEquals(expectedLCS, actualLCS);
    }
```

none of these options worked. Any idea why?
It seems like the @Test has to run for the manual branch coverage to be used.